python/dgl/graph.py

```diff
@@ -936,6 +941,20 @@ def __init__(self,
         self._apply_node_func = None
         self._apply_edge_func = None
 
+        # batched graph
+        self._batch_size = batch_size
+        self._batch_num_nodes = batch_num_nodes
+        self._batch_num_edges = batch_num_edges
+
+        # subgraph
+        self._parent = parent
+        self._parent_nid = None
+        self._parent_eid = None
+        self._subgraph_index = sgi
+        if sgi is not None:
+            self._parent_nid = sgi.induced_nodes
+            self._parent_eid = sgi.induced_edges
```

Comment:
Shouldn't need `self._parent_nid`, `self._parent_eid` and `self._subgraph_index` members. Mappings are saved under `dgl.NID` and `dgl.EID` fields. Please refer to the implementation of `dgl.HeteroGraph.node_subgraph`.

Reply:
DONE.